charts/bootstrap_tm_prerequisites/templates/argo.yaml

```
@@ -22,8 +22,8 @@ data:
         artifactRepository:
             s3:
                 bucket: {{.Values.objectStorage.bucketName}}
-                keyPrefix: testmachinery     
-                endpoint: {{.Values.objectStorage.serviceEndpoint}}:{{.Values.objectStorage.port}}        #AWS => s3.amazonaws.com; GCS => storage.googleapis.com
+                keyPrefix: testmachinery
+                endpoint: {{.Values.objectStorage.serviceEndpoint}}.{{.Release.Namespace}}:{{.Values.objectStorage.port}}        #AWS => s3.amazonaws.com; GCS => storage.googleapis.com
```

Maybe amend the url (avoid relying on /etc/resolv.conf to resolve correctly), see also https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#services

```suggestion
                endpoint: {{.Values.objectStorage.serviceEndpoint}}.{{.Release.Namespace}}.svc.cluster.local:{{.Values.objectStorage.port}}        #AWS => s3.amazonaws.com; GCS => storage.googleapis.com
```